charts/polkadot-watcher-transaction/values.yaml

```diff
@@ -2,26 +2,22 @@ environment: production
 
 image:
   repo: web3f/polkadot-watcher-transaction
-  tag: v0.4.14
+  tag: v0.5.0
 
 config:
   endpoint: "wss://kusama-rpc.polkadot.io/"
   port: 3000
   logLevel: info
   matrixbot:
     endpoint: "http://matrixbot:8080/skill/eventmanager/webhook"
-    strategy: "NoDuplicates"
-    noDuplicatesWindow: 5000
   subscriber:
     modules:
-      transferEvent:
-        enabled: true
-        sent: true
-        received: true
-      balanceChange:
-        enabled: true
-      transferExtrinsic:
-        enabled: false
+      transferEventScanner:
+      enabled: true
+      sent: true
+      received: true
+      dataDir: "/app/data"
+      scanInterval: 600000
```

Comment:
I'd recommend to rename scanInterval field to scanIntervalSecs or some unit that people directly understand what unit those 600000 mean.

Reply:
Here iiuc a good name would be scanIntervalMilliSecs